vignettes/train-and-deploy-with-caret/train-and-deploy-with-caret.Rmd

```diff
@@ -0,0 +1,284 @@
+
+## Launch RStudio and load the azureml package
+
+We recommend using RStudio to run these examples. Launch RStudio and then:
+
+```{r launch}
+library(azuremlsdk)
+```
+
+You'll use these additional R packages:
+
+```{r packages, eval=FALSE}
+install.packages("DAAG")
+install.packages("caret")
+```
+
+## Load your workspace
+
+Now define a workspace object in R by loading your config file.
+
+```{r load_workpace, eval=FALSE}
+ws <- load_workspace_from_config()
+```
+
+## Load data and prepare for training 
+
+This tutorial uses data from the [DAAG package](https://cran.r-project.org/package=DAAG). This dataset includes data from over 25,000 car crashes in the US, with variables you can use to predict the likelihood of a fatality. First, import the data into R and transform it into a new dataframe `accidents` for analysis, and export it to an `Rdata` file.
+
+```{r load_data, eval=FALSE}
+library(DAAG)
+data(nassCDS)
+
+accidents <- na.omit(nassCDS[,c("dead","dvcat","seatbelt","frontal","sex","ageOFocc","yearVeh","airbag","occRole")])
+accidents$frontal <- factor(accidents$frontal, labels=c("notfrontal","frontal"))
+accidents$occRole <- factor(accidents$occRole)
+
+saveRDS(accidents, file="accidents.Rd")
+```
+
+## Upload data to the datastore
+
+Azure Machine Learning workspaces provide a default datastore where you can store data and other files needed for analysis.
+Here, upload the accidents data you created above to the datastore.
+
+```{r upload_data, eval=FALSE}
+ds <- get_default_datastore(ws)
+target_path <- "accidentdata"
+upload_files_to_datastore(ds,
+                          list("./accidents.Rd"),
+                          target_path = target_path,
+                          overwrite = TRUE)
+```
+
+## Create a compute resource
+
+When you need more power than your local laptop to train a model, create a compute resource. 
+Here you create a virtual machine in Azure (and give it a name of `rcluster`) to use for training your model.
+
+```{r create_cluster, eval=FALSE}
+cluster_name <- "rcluster"
+compute_target <- get_compute(ws, cluster_name = cluster_name)
+if (is.null(compute_target)) {
+  vm_size <- "STANDARD_D2_V2" 
+  compute_target <- create_aml_compute(workspace = ws,
+                                       cluster_name = cluster_name,
+                                       vm_size = vm_size,
+                                       max_nodes = 1)
+}
+```
+
+## Train a model
+
+Now fit a logistic regression model on your uploaded data using your remote compute target.
+
+> NOTE
+> You may need to wait a few minutes for your compute cluster to be provisioned before moving on to the next step.
+
+The script to fit the model is called `accidents.R`. It will be run as a command-line script with `Rscript`.   `Rscript` takes one argument, `-d` to specify the storage folder where the data file is located. The argument will be provided for you by Azure Machine Learning.
```

Comment:
Maybe add. Provided by Azure Machine Learning..... datastore target path properties (also add the hyperlink to the API in R SDK)